src/web/services/masquerade-mode.service.spec.ts

```diff
@@ -1,17 +1,40 @@
 import { TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
 
 import { MasqueradeModeService } from './masquerade-mode.service';
 
 describe('MasqueradeModeService', () => {
-  beforeEach(() => TestBed.configureTestingModule({
-    imports: [
-      RouterTestingModule,
-    ],
-  }));
+  let service: MasqueradeModeService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        MasqueradeModeService,
+      ],
+    });
+    service = TestBed.get(MasqueradeModeService);
+  });
 
   it('should be created', () => {
-    const service: MasqueradeModeService = TestBed.get(MasqueradeModeService);
     expect(service).toBeTruthy();
   });
+
+  it('getMasqueradeUser should be defined', () => {
```

Comment:
I think the “it” is supposed to be part of the sentence, so it should be something like `it(‘should be defined’, ...)`

Reply:
Nice, thanks for the info! :)